lib/iex/endpoints/search.rb

```diff
@@ -0,0 +1,20 @@
+module IEX
+  module Endpoints
+    module Search
+      def search(fragment, options = {})
+        # consider raising if fragment.blank?
+        get([
+          'search',
+          fragment
+        ].compact.join('/'), options).map do |data|
+          # Per the IEX documentation, any type of Resource could be returned
```

Comment:
Yes, please. And create a base resource for anything unsupported.

Reply:
fyi, they have not released this feature yet, the docs just mention that it is coming

Reply:
I don't see a problem supporting something in pre-release, but we have to say it in README and point to whatever IEX docs that say the same